android sdk 2.2.1 rns added (#7063)
harness/developer-hub: release-notes/feature-flags.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

#### Last updated: June 14, 2024

## June 2024

### Android SDK

#### Version 2.2.1

**New features and enhancements**:

>  - Added a new method for closing the SDK. The `closeWithFuture()` method can be used to ensure the SDK has been closed before continuing, for example, re-initializing the SDK. (FFM-11625)

Reviewer: @jenniferopal There's a Zendesk ticket we should add for reference.
[64818](https://harnesssupport.zendesk.com/agent/tickets/64818)

Author: HI @brian-f-harness , I missed that. I'll add the ticket to my next PR where I'm adding the release notes for Flutter SDK.

>  - Adds a new method for closing the SDK. The `closeWithFuture()` method can be used if you need to ensure the SDK has been closed before continuing e.g. re-initialising the SDK. (FFM-11658)

Reviewer: ```suggestion
 - Added a new method for closing the SDK. The `closeWithFuture()` method can be used to ensure the SDK has been closed before continuing, for example, re-initializing the SDK. (FFM-11658)
```

Reviewer: The FFM-11658 ticket is for your release notes work. Is there a Jira associated with the new feature you can use here for reference instead?

Author: yeah, there was. I typed the wrong jira number. my mistake.
